pkg/cmd/types.go

```diff
@@ -77,11 +77,12 @@ type SecretConvertor interface {
 
 //ServiceConfigs are collection of configurations for services in a specific namespace
 type ServiceConfigs struct {
-	Version     int              `json:"version"`
-	ClusterName string           `json:"clusterName"`
-	Namespace   string           `json:"namespace"`
-	ClientID    string           `json:"clientId,omitempty"`
-	Services    []*ServiceConfig `json:"services"`
+	Version     int                       `json:"version"`
+	ClusterName string                    `json:"clusterName"`
+	Namespace   string                    `json:"namespace"`
+	ClientID    string                    `json:"clientId,omitempty"`
+	Services    []*ServiceConfig          `json:"services"`
+	Https       []*CertificatePinningHash `json:"https,omitempty"`
```

Comment:
to allow extend `https` object in the future, let's change it to an object, and add a key for certificate pinnings.